### ISSUE TITLE
Git not working

### ISSUE THREAD

**Issue by vyanary11** [microsoft/vscode#136920]
Issue Type: <b>Bug</b>

i just pull my project and it just loading like not responding

VS Code version: Code 1.62.1 (f4af3cbf5a99787542e2a30fe1fd37cd644cc31f, 2021-11-05T10:57:55.946Z)
OS version: Windows_NT x64 10.0.19042
Restricted Mode: No
Remote OS version: Linux x64 5.4.72-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-6198DU CPU @ 2.30GHz (4 x 2400)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.90GB (1.37GB free)|
|Process Argv|--remote=wsl+Ubuntu --crash-reporter-id cd67e847-542d-4fdd-889d-d9e087c31cff|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu|
|OS|Linux x64 5.4.72-microsoft-standard-WSL2|
|CPUs|Intel(R) Core(TM) i5-6198DU CPU @ 2.30GHz (2 x 2399)|
|Memory (System)|1.45GB (0.21GB free)|
|VM|0%|
</details><details><summary>Extensions (22)</summary>

Extension|Author (truncated)|Version
---|---|---
vue|jcb|0.1.5
dotenv|mik|1.0.1
remote-wsl|ms-|0.58.5
awesome-flutter-snippets|Nas|3.0.2
laravel5-snippets|one|1.13.0
bootstrap4-vscode|the|6.1.0
twig|wha|1.0.2
http-client|aar|1.0.5
laravel-extra-intellisense|ami|0.6.1
vscode-intelephense-client|bme|1.7.1
laravel-goto-view|cod|1.3.4
vscode-mysql-client2|cwe|4.1.8
EditorConfig|Edi|0.16.4
php-debug|fel|1.22.0
php-intellisense|fel|2.3.14
dotenv|mik|1.0.1
vscode-duplicate|mrm|1.2.1
laravel-blade|one|1.28.0
laravel-extension-pack|one|1.0.0
laravel-artisan|rya|0.0.28
laravel-goto-controller|ste|0.0.15
bootstrap4-vscode|the|6.1.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonvspyt551cf:30345471
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
vsccppwt:30382697
bridge0708:30335490
dockerwalkthru:30377721
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30391933
pythonvspyt187:30373474
vsqsis300:30386381
vsaa593:30376534
vssld246:30386377

```

</details>

**Comment by vyanary11** [comment]
![image](https://user-images.githubusercontent.com/12364093/141226348-85e37516-68e6-462e-bddd-5a943a11f312.png)

**Comment by gjsjohnmurray** [comment]
/duplicate - see https://github.com/microsoft/vscode/issues/136873#issuecomment-965402902 for advice.

Will be fixed in upcoming 1.62.2
